scripts/install_ansible_debops_controller.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+################################################################################
+################################################################################
+#
+# https://docs.debops.org/en/master/user-guide/install.html
+#
+# This scripts installs DebOps and its dependencies on a (fresh) system.
+# It creates a dedicated DebOps user (e.g. ansible) who should then execute all DebOps-playbooks.
+# This script only installs DebOps globally , it does not initiliaze any DebOps-project.
+#
+################################################################################
+
+set -o nounset -o pipefail -o errexit
+
+# The username used by debops
+DEBOPS_ADMIN_USER="${DEBOPS_ADMIN_USER:-ansible}"
+
+echo "Install dependencies"
+apt-get -q update
+DEBIAN_FRONTEND=noninteractive apt-get \
+  --no-install-recommends -yq install \
+  iproute2 \
+  levee \
+  openssh-client \
+  python3-apt \
+  python3-distro \
+  python3-dnspython \
+  python3-future \
+  python3-ldap \
+  python3-openssl \
+  python3-pip \
+  python3-wheel \
+  python3-setuptools \
+  procps \
+  sudo \
+  tree \
+  git \
+  gpgv2 \
+  uuid-runtime \
+  sshpass \
+  encfs \
+  dirmngr \
+  rsync \
+  pass
+
+echo "Install ansible and debops"
+pip3 install \
+ debops[ansible]
+
+echo "Cleaning up cache directories..."
+apt clean
+rm -rf /var/lib/apt/lists/* /var/cache/apt/archives/*.deb /root/.cache/*
```

Comment:
This probably is not needed, and will actually force the user to run `apt-get update` again to get fresh information about APT repositories. That should not be included here.

Reply:
at least apt get clean should remain